docs/userGuide/contentAuthoring.md

```diff
@@ -375,6 +375,50 @@ Front matter can also be included from a separate file, just like how content ca
 
 This will result in `index.md` having the title `Binary Search Tree` and the specified keywords in order for it to be looked up through the search bar.
 
+### Inserting content into a page's head element
+
+While authoring your website, you may want to have your own CSS or Javascript file to be included in a page.
+
+- Start by creating a head file with Markdown extension (`.md`) in the `_markbind/head` folder.
+    - More than one head file can be created for different pages.
+
+- Author your `<style>` elements for CSS files and `<link>` elements for Javascript files using HTML as shown below.
+    - Ensure that any url starts from the root directory <code>{<span></span>{baseUrl}}/</code> when you are referencing your files.

```

Comment:
`Ensure that any url starts...` -> `Ensure that your URLs start...`
`... from the root directory {{baseUrl}} ...` -> `... from the root directory, by using {{baseUrl}} ...`